doc/usecases/cli/UC_namespace.md

```diff
@@ -0,0 +1,20 @@
+# Use Case: cli extract namespace
+
+## Summary
+
+- **Scope:** CLI use
+- **Level:** Developer Goal
+- **Actors:** CLI user
+- **Brief:** Extract the namespace from a given keyname
+
+## Scenarios
+
+- **Precondition:** -
+- **Main success scenario:**
+  - The namespace fo a keyname is printed
```

Comment:
What must be done before that step?
Please describe what the CLI user has to do before the namespace is printed.

```suggestion
  - The namespace of a keyname is printed
```